cmd/empire/main.go

```diff
@@ -27,9 +27,10 @@ const (
 	FlagDBPath = "path"
 	FlagDB     = "db"
 
-	FlagDockerSocket = "docker.socket"
-	FlagDockerCert   = "docker.cert"
-	FlagDockerAuth   = "docker.auth"
+	FlagDockerSocket    = "docker.socket"
+	FlagDockerCert      = "docker.cert"
+	FlagDockerAuth      = "docker.auth"
+	FlagDockerLogDriver = "docker.logdriver"
```

Comment:
I think this should actually live under the `ecs.*` namespace. The `docker.*` flags control options for connecting to the local docker daemon.